.github/workflows/main.yml

```diff
@@ -1,23 +1,31 @@
-name: 'CI Multiplatform Build'
+name: 'BUILD'
 
 on:
   push:
-    paths-ignore:
-      - '**.md'
-    branches:
-      - master
-    tags:
-      - '*'
+    paths-ignore: ['**.md']
+    branches: [master]
+    tags: ['*']
   pull_request:
-    paths-ignore:
-      - '**.md'
-    branches:
-      - master
+    paths-ignore: ['**.md']
+    branches: [master]
 
 env:
-  MACOS_CEF_BUILD_VERSION: '4638'
-  LINUX_CEF_BUILD_VERSION: '4638'
-  WINDOWS_CEF_BUILD_VERSION: '4638'
+  CEF_BUILD_VERSION_MAC: '4638'
+  CEF_HASH_MAC_X86_64: '309a58607f9f910a54f3e2e38beb255abb67823b47e4224895c35812ed0e8f77'
+  CEF_HASH_MAC_ARM64: '510687954ece8f109116f1ac8c915d4114c9632193b288bc7db7380b70887ab8'
```

Comment:
I'm getting a failed hash check with CEF when building on an M1.
```
+ cef_binary_4638_macos_arm64.tar.xz downloaded successfully and failed hash check
```
It seems that while the hash `510687954ece8f109116f1ac8c915d4114c9632193b288bc7db7380b70887ab8` was expected, the hash `27809aac427b9c97b0dadcab04371c5f8211f36c6f3e1caa00822de89c5ed249` was found instead.

Reply:
The commit hash check fails because we've updated the CEF package since the last time this PR was updated.  We aren't planning to further update/rebase this PR until after OBS Studio 27.2.0 releases at the earliest.